src/screens/profile/container/profileContainer.js

```diff
@@ -84,15 +85,13 @@ class ProfileContainer extends Component {
       return;
     }
 
-    if (currentUsername) {
-      this._loadProfile(currentUsername);
-    }
-
     if (
-      activeBottomTab !== nextProps.activeBottomTab &&
-      nextProps.activeBottomTab === 'ProfileTabbar'
+      (activeBottomTab !== nextProps.activeBottomTab &&
+        nextProps.activeBottomTab === ROUTES.TABBAR.PROFILE) ||
```

Comment:
use get from lodash

Reply:
it has different condition, `get` not suitable for use in here